### ISSUE TITLE
Do you still accept pull requests to this repository?

### ISSUE THREAD

**Issue by 8ctopus** [pyrocms/lex#19]

**Comment by WanWizard** [comment]
Looks like it is abandonded. It throws deprecated messages in PHP 8.2.

**Comment by 8ctopus** [comment]
Yes indeed, I forked it and fixed the issues I found: https://packagist.org/packages/8ctopus/lex

**Comment by WanWizard** [comment]
As a one-off, or are you planning on maintaining it? If not we'll fork as well.

We have a few apps that rely on Lex for email templating, so we need a maintained repo.

**Comment by 8ctopus** [comment]
we're using it too, so it will be maintained for 2023, better spend the energy together.
